friends/src/App.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import axios from 'axios';
+import Friend from './Friend';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      friends: []
+    }
+  }
+
+  componentDidMount() {
+    axios.get('http://localhost:5000/friends')
+    .then(req => {
+      this.setState({ friends: req.data })
+    })
+    .catch(err => {console.log(err)})
+  }
+
+  render() {
+    return (
+      <div className="App">
+        {this.state.friends.map(input => <Friend key={input.id} data={input} />)}
```

Comment:
Everything will still render obviously, but I'd recommend altering your naming conventions to be more meaningful -- you iterate over each `friend` in your `friends` array, pass it in as `friend={friend}`, and inside the component access the info with `friend.name`, `friend.age`, etc. Naming conventions like this will scale better, prevent possible confusion for you when referencing data across different components, and make for easier reading for others going through your code